### PR TITLE
Update cross-site instructions

### LINKED ISSUE #1564: Update information for cross-site replication
### Description

RHDG 7.3 is no longer valid, so the instructions will be removed.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #1564

@tkyjovsk, @mposolda, @pskopek Can you verify this change?  We are no longer supporting RHDG 7.3.x for cross-site replication for RH SSO 7.6.

### CODE REVIEW

**Comment by andymunro** [comment]
@iankko Can you approve this?

File: server_installation/topics/operating-mode/crossdc.adoc [hunk 1]
```diff
@@ -679,7 +194,7 @@ endif::[]
 
 WARNING: These steps usually need to be done for all the {project_name} caches mentioned in <<backups>>.
 
-**Automatically** - After some amount of failed backups, the `site2` will usually be taken offline automatically. This is done due the configuration of `take-offline` element inside the cache configuration as configured in <<jdgsetup>>.
+**Automatically** - After some amount of failed backups, the `site2` will usually be taken offline automatically. This is done due the configuration of `take-offline` element inside the cache configuration.
```
Comment (tkyjovsk) [inline server_installation/topics/operating-mode/crossdc.adoc]: amount --> number

File: server_installation/topics/operating-mode/crossdc.adoc [hunk 2]
```diff
@@ -817,11 +332,9 @@ Note the `mode` attribute of cache-configuration element.
 
 The following tips are intended to assist you should you need to troubleshoot:
 
-* It is recommended to go through the <<setup>> and have this one working first, so that you have some understanding of how things work. It is also wise to read this entire document to have some understanding of things.
-
-* Check in jconsole cluster status (GMS) and the JGroups status (RELAY) of {jdgserver_name} as described in <<jdgsetup>>. If things do not look as expected, then the issue is likely in the setup of {jdgserver_name} servers.
+* We recommended that you go through the procedure for xref:assembly-setting-up-crossdc[Setting up cross-site replication] and have this one working first, so that you have some understanding of how things work. It is also wise to read this entire document to have some understanding of things.
```
Comment (tkyjovsk) [inline server_installation/topics/operating-mode/crossdc.adoc]: recommended --> recommend

**Review by tkyjovsk — CHANGES_REQUESTED** [review]

File: server_installation/topics/operating-mode/crossdc.adoc (hunk 1 above)
Comment (iankko) [reply server_installation/topics/operating-mode/crossdc.adoc]: > amount --> number

@andymunro +1 on this. Per [Cambridge dictionary suggestion](https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of) `amount of` is used for uncountable nouns, while `number of` for countable ones. Count of failed backups looks like a countable one.

Maybe replace with `After a certain number of failed backups....`?

**Review by iankko — COMMENTED** [review]
One small nitpick, but in general LGTM!
@andymunro Thanks, Andy!

**Comment by andymunro** [comment]
The broken links are to 7.6 URLs that don't exist yet.

**Comment by andymunro** [comment]
Thanks, @iankko, @tkyjovsk.  Please approve if it is okay now.

**Review by tkyjovsk — APPROVED** [review]

**Review by tkyjovsk — APPROVED** [review]

**Review by iankko — APPROVED** [review]
LGTM @andymunro Thanks, Andy!

**Pull request merged**